Author: Diane Trout <diane@ghic.org>
Description: Add __init__ file to make kolab a valid python package
 The comment in __init__.py is just so quilt makes the file,
 it could be an empty file if included in git.
Index: libkolab/CMakeLists.txt
===================================================================
--- libkolab.orig/CMakeLists.txt	2015-11-20 21:17:36.405667632 +0100
+++ libkolab/CMakeLists.txt	2015-11-20 21:17:36.401667609 +0100
@@ -202,6 +202,11 @@
 if(PYTHON_BINDINGS)
     generatePythonBindings(shared shared.i)
     add_subdirectory(kolabformat/python)
+    install( FILES
+        __init__.py
+        DESTINATION ${PYTHON_INSTALL_DIR}/kolab
+    )
+
 endif(PYTHON_BINDINGS)
 
 if(PHP_BINDINGS)
Index: libkolab/__init__.py
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ libkolab/__init__.py	2015-11-20 21:17:36.401667609 +0100
@@ -0,0 +1,2 @@
+# Needed to make python package
+
